droptools-example/src/main/java/com/bendb/example/resources/PostsResource.java

```diff
@@ -44,7 +44,7 @@ public List<BlogPost> findPostsByTag(@QueryParam("tag") String tag, @JooqInject(
                 .leftOuterJoin(pt)
                 .on(BLOG_POST.ID.equal(pt.POST_ID))
                 .whereExists(selectOne()
-                        .from(tableByName("postIds"))
+                        .from(POST_TAG)
```

Comment:
This change looks to me like it changes the semantics from "select blog posts whose IDs are in the CTE" to "select blog posts that have tag".  How did you determine that this was a) necessary, and b) correct?

Reply:
I see what you mean, I think this updated line is the correct fix. tableByName was no longer working, thats why I went about replacing it.